src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -5,8 +5,6 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              xmlns:ui="clr-namespace:Dynamo.UI"
              xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
-             xmlns:viewmodels="clr-namespace:Dynamo.ViewModels" 
-             d:DataContext="{d:DesignInstance Type=viewmodels:InstalledPackagesViewModel}"
```

Comment:
The installed packages view ultimately gets its `DataContext `set to `PreferencesViewModel `despite this so I just removed it to avoid confusion.